Extensions/xkit_patches.js

```diff
@@ -129,6 +129,52 @@ XKit.extensions.xkit_patches = new Object({
 	patches: {
 		"7.9.0": function() {
 
+			XKit.svc = {
+				blog: {
+					followed_by: data => new Promise((resolve, reject) => {
+						XKit.tools.Nx_XHR({
+							method: "GET",
+							url: "https://www.tumblr.com/svc/blog/followed_by?" + $.param(data),
+							onload: resolve,
+							onerror: reject
+						});
+					})
+				},
+
+				conversations: {
+					participant_info: data => new Promise((resolve, reject) => {
+						XKit.tools.Nx_XHR({
+							method: "GET",
+							url: "https://www.tumblr.com/svc/conversations/participant_info?" + $.param(data),
+							onload: resolve,
+							onerror: reject
+						});
+					})
+				}
+			};
+
+			/**
+			 * Determines whether a user is following the given blog.
+			 * The logged-in user must be a member of the given blog to determine this.
+			 * @param {String} username
+			 * @param {String} blog
+			 * @return {Promise<Boolean>}
+			 */
+			XKit.interface.is_following = function(username, blog) {
+				return new Promise(resolve => {
+					XKit.svc.conversations.participant_info({
+						"q": username,
+						"participant": blog
+					})
+					.then(response => resolve(response.responseObj.response.is_blog_following_you))
```

Comment:
just return the value

Reply:
returning the value makes `is_following` return that though? which we don't want unless we want to also rewrite how that function is used across extensions